pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/metrics/ManagedLedgerMetrics.java

```diff
@@ -108,6 +108,8 @@ private List<Metrics> aggregate(Map<Metrics, List<ManagedLedgerImpl>> ledgersByD
                         (double) lStats.getReadEntriesErrors());
                 populateAggregationMapWithSum(tempAggregatedMetricsMap, "brk_ml_ReadEntriesRate",
                         lStats.getReadEntriesRate());
+                populateAggregationMapWithSum(tempAggregatedMetricsMap, "brk_ml_ReadEntriesOpsCacheMissesRate",
+                        lStats.getReadEntriesOpsCacheMissesRate());
```

Comment:
Why didn't this go with the other cache metrics?

https://github.com/apache/pulsar/blob/82237d3684fe506bcb6426b3b23f413422e6e4fb/pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/metrics/ManagedLedgerCacheMetrics.java#L49-L58

Reply:
Note that these cache metrics already have a well known format.

Reply:
> Why didn't this go with the other cache metrics?
> 

I think that it's more reasonable that here It follows the style of other `ManagedLedgerMetrics`, like `brk_ml_ReadEntriesRate` and so on.  They all are aggregation by namespace, not aggregation by ledger.

In other place we go with the well know format, like `pulsar_storage_read_cache_misses_rate` and `pulsar_broker_storage_read_cache_misses_rate`

https://github.com/apache/pulsar/pull/20001/files#diff-79bde2a6feed80b6b6b4b6950bc7d4ccd93946a8ec0b40cc1bef2e7b3e498f21R155-R157